src/graphics/Window.h

```diff
@@ -0,0 +1,117 @@
+//
+// Created by ajer on 11.02.19.
+//
+
+#pragma once
+
+#include <GLFW/glfw3.h>
+
+
+namespace Bald::Graphics {
+
+    /**
+     * @class Window
+     * @brief Implementation of simple Window
+     */
+    class Window {
+    public:
+        /** ========== METHODS ========== */
+
+        /**
+         * @fn                      Window
+         * @param [std::string]     title
+         * @param [int]             width
+         * @param [int]             height
+         * @brief                   constructor for the window
+         */
+        explicit Window(const char *title = "Window", int width = 800, int height = 600,
+                        bool closed = false)
```

Comment:
Shouldn't closed be true by default? When ::Init will be invoked and everything will be run correctly than we can certainly tell that window isNotClosed.

Reply:
fixed